extractor.go

```diff
@@ -127,9 +127,12 @@ func (extr *ContentExtractor) GetMetaLanguage(document *goquery.Document) string
 		language = attr[0:idx]
 	}
 
-	_, ok := sw[language]
+	lang := extr.config.stopWords.GetStopWordsByLanguage(language)
 
-	if language == "" || !ok {
+	//_, ok := sw[language]
+
+	//if language == "" || !ok {
+	if language == "" || lang == nil {
```

Comment:
[nit] the language == "" check is redundant here, since lang would already be nil if that were the case.

Reply:
the code here is a bit muddled, did a bit of cleanup

Reply:
ty!